guide/english/ruby/common-array-methods/index.md

```diff
@@ -185,22 +185,15 @@ end
 ```
 
 #### .map
-The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements.
+The .map method is the same as the .collect method. The .map and .collect methods iterate over each element of the array, allowing you to perform actions on them. The .map and .collect methods differ from the .each method in that they return an array containing the transformed elements:
 
 ``` ruby
 array.map { |element| element * 2 }
-  puts element
-end
-=> 
-0
-2
-4
-6
-8
+=> [0, 2, 4, 6, 8]
```

Comment:
same as the suggestion from the previous snippet, it is necessary to declare the array first